app/controllers/v1/sentences_controller.rb

```diff
@@ -98,9 +122,82 @@ def build_response(data)
       }
     end
 
-    # エラーレスポンス
-    def render_error_response(error)
-      render json: { error: }, status: :not_found
+    # createの補助メソッド
+
+    # 必須パラメータの存在を確認
+    def check_required_params
+      unless params[:parent_sentence_id].blank? || params[:sentence].blank?
+        return params[:parent_sentence_id], params[:sentence]
+      end
+
+      render_error_response(422, '必須項目に空欄があります。')
+    end
+
+    # 親投稿の存在を確認
+    def find_parent_sentence(parent_sentence_id)
+      parent_sentence = Sentence.find_by(sentence_id: parent_sentence_id)
+      render_error_response(422, '親投稿が見つかりません') if parent_sentence.nil?
```

Comment:
@technokuro 
こちらとその下に書いたその他のエラー（「投稿の追加に失敗しました」）は自分の感覚でも400かなと感じて、打ち替えてみたのですが、下記のrspecのエラーによりテストがパスできなかったため、一旦422のままにしております。ご確認お願いしますmm

「レスポンスのステータス コードが :unprocessable_entity (422) であると予想しましたが、:bad_request (400) でした」という内容
```
% docker-compose exec backend bundle exec rspec
/usr/local/bundle/gems/bootsnap-1.18.4/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30: warning: /usr/local/lib/ruby/3.3.0/drb.rb was loaded from the standard library, but will no longer be part of the default gems since Ruby 3.4.0. Add drb to your Gemfile or gemspec. Also contact author of activesupport-7.0.8.4 to add drb into its gemspec.
....................FF...

Failures:

  1) Sentences POST /v1/sentences when required parameters are missing returns an unprocessable entity status
     Failure/Error: expect(response).to have_http_status(:unprocessable_entity)
       expected the response to have status code :unprocessable_entity (422) but it was :bad_request (400)
     # ./spec/requests/sentences_spec.rb:86:in `block (4 levels) in <main>'

  2) Sentences POST /v1/sentences when parent_sentence_id does not exist returns an unprocessable entity status
     Failure/Error: expect(response).to have_http_status(:unprocessable_entity)
       expected the response to have status code :unprocessable_entity (422) but it was :bad_request (400)
     # ./spec/requests/sentences_spec.rb:96:in `block (4 levels) in <main>'

Finished in 1.41 seconds (files took 1.07 seconds to load)
25 examples, 2 failures

Failed examples:

rspec ./spec/requests/sentences_spec.rb:84 # Sentences POST /v1/sentences when required parameters are missing returns an unprocessable entity status
rspec ./spec/requests/sentences_spec.rb:94 # Sentences POST /v1/sentences when parent_sentence_id does not exist returns an unprocessable entity status
```